ui/src/app/certificate_requests/row.tsx

```diff
@@ -0,0 +1,32 @@
+const extractCSR = (csrPemString: string) => {
+    //TODO
+}
+
+const extractCert = (certPemString: string) => {
+    //TODO
+}
+
+type rowProps = {
+    id: number,
+    csr: string,
+    certificate: string
+}
+export default function Row({ id, csr, certificate }: rowProps) {
+    return (
+        <tr>
+            <td className="" data-test-column="id">{id}</td>
+            <td className="u-overflow--visible" data-test-column="details">
+                <p>CN: example.com</p>
+                <p>SAN: example.com, 127.0.0.1, 1.2.3.4.5.56</p>
```

Comment:
Would bullets show up better as a comma separated list?

Reply:
What do you mean by that? having the SAN's be <ul></ul> elements?

Reply:
Would this:
```
Subject Alternative Names:
- example.com
- 127.0.0.1
- 1.2.3.4.5.56
```
Be better than this?

```
Subject Alternative Names: example.com, 127.0.0.1, 1.2.3.4.5.56
```

Reply:
I think with that we would only be able to fit 2-3 CSR's on one page, which is too little imo (I use all my chat apps with the whitespace removed :D, so it is in partly an aesthetics and partly a functional change). It could work better if we hide this behind an expandable row like [so](https://vanillaframework.io/docs/base/tables#expanding), but then we'd still need to populate this field with a thumbnail info.

One of the reasons I like the `, ` approach is it also makes it viable to ctrl+f through the page with multiple SAN's by just putting in `example.com, 12.` or something like that.